### PR TITLE
updating readme instructions for postgres

### PR DESCRIPTION
**Pull request by beaesguerra**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -145,13 +145,33 @@ Custom tags can be added to the tests for filtering in the future.
...
-. Run the initialization commands for creating the database tables, `\i db-init.sql`, the shell must be started in the same place as `db-init.sql` for this step.
+. Create a user with correct permissions to create a database. The recommended default username is `boresha`.
+
+	CREATE ROLE boresha WITH LOGIN PASSWORD '<pw>';
+	ALTER ROLE boresha CREATEDB;
+
+. Using the user you just created, create a database and grant the proper permissions to the user. The recommended default name for the database is `agricoredev`.
+
+	psql -U boresha
+	CREATE DATABASE agricoredev;
```
Comment (ExtraGravity) [inline README.adoc]: Should be `agricore_dev` to match the example.

File: README.adoc [hunk 2]
```diff
@@ -145,13 +145,33 @@ Custom tags can be added to the tests for filtering in the future.
...
+To set up a database for the Agricore, follow the steps below. Please start from the root directory of the project (ie. agricore/).
 
 . The Agricore uses PostgreSQL 10.1, although any version >=9.1 should work as well.
-. Create database with correct user permissions, the recommended default names are `agricore-dev` for the database name, and `boresha` for the user.
-. Run the `psql` shell using psql -d <databasename> -U <username>.
-. Install the `uuid-ossp` extension, `CREATE EXTENSION "uuid-ossp"`, this is already packaged but not loaded for postgres >=9.1.
-. Run the initialization commands for creating the database tables, `\i db-init.sql`, the shell must be started in the same place as `db-init.sql` for this step.
+. Create a user with correct permissions to create a database. The recommended default username is `boresha`.
+
+	CREATE ROLE boresha WITH LOGIN PASSWORD '<pw>';
```
Comment (ExtraGravity) [inline README.adoc]: You can put these in code blocks.

```asciidoc
[source,postgres]
----
CREATE ROLE...
----
```

**Review by ExtraGravity — CHANGES_REQUESTED** [review]

File: README.adoc (hunk 2 above)
Comment (beaesguerra) [reply README.adoc]: fixed

File: README.adoc (hunk 1 above)
Comment (beaesguerra) [reply README.adoc]: fixed

**Review by ExtraGravity — APPROVED** [review]

**Pull request merged**